api_docs/include/rest-endpoints.md

```diff
@@ -14,7 +14,9 @@
 * [Get a message's edit history](/api/get-message-history)
 * [Update personal message flags](/api/update-message-flags)
 * [Update personal message flags for narrow](/api/update-message-flags-for-narrow)
-* [Mark messages as read in bulk](/api/mark-all-as-read)
+* [Mark all messages as read](/api/mark-all-as-read)
+* [Mark messages in a stream as read](/api/mark-stream-as-read)
+* [Mark messages in a topic as read](/api/mark-topic-as-read)
```

Comment:
We also have an open issue to remove all three of these endpoints in favor of the new endpoint created for "mark unread".

Reply:
That’s #23598. (But that’s likely to take a while and shouldn’t block this fix.)

Reply:
Yep agreed.